school/models/school.py

```diff
@@ -235,14 +235,12 @@ def onchange_combine(self):
         self.name = str(self.standard_id.name
                         ) + '-' + str(self.division_id.name)
 
-    @api.multi
```

Comment:
reason for removing multi ?

Reply:
We don't need decorator @api.multi here, as only @api.depends will do the job. I have referred base code and removed.